javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,102 @@
+console.log('JS1 WEEK4');
+
+const listOftodos = []
+const storeName = []
+const calcSentence = []
+function getReply(command) {
+
+   if (command.includes('Hello my name is') || command.includes('my name is')){
+    const splitNameSentence = command.split(' ')
+    const rememberName = splitNameSentence.length <= 4 ? splitNameSentence[3] : splitNameSentence[4];
+    storeName.push(rememberName)
+    console.log('Nice to meet you ' + rememberName);
+   } 
```

Comment:
You can also use template literals for a shorter and more efficient code. What if the name in the user's input is not exactly at the index you thought it would be? 🤔